tests/drivers/clock_control/pwm_clock/boards/xmc47_relax_kit.overlay

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) 2023 Andriy Gelman
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/ {
+	pwmclock: pwmclock {
+		status = "okay";
+		compatible = "pwm-clock";
+		#clock-cells = <1>;
+		clock-frequency = <1000000>;
+		pwms = <&pwm_ccu80 0 PWM_HZ(1000000) PWM_POLARITY_NORMAL>;
```

Comment:
you could use `PWM_KHZ` or `PWM_MHZ` here

Reply:
thanks, I changed to PWM_KHZ(). Also rebased to fix merge conflicts.